src/main/java/org/jenkinsci/plugins/codedx/CodeDxBuildAction.java

```diff
@@ -5,25 +5,30 @@
 import java.io.Serializable;
 import java.util.*;
 
+import hudson.model.Run;
+import jenkins.tasks.SimpleBuildStep;
 import org.jenkinsci.plugins.codedx.model.CodeDxReportStatistics;
 import org.kohsuke.stapler.StaplerProxy;
 
 /**
  *
  * @author ademartini This file is heavily derived from the sloccount-plugin (author: lordofthepigs)
  */
-public class CodeDxBuildAction implements Action, Serializable, StaplerProxy {
-	/** Serial version UID. */
-	private static final long serialVersionUID = 0L;
+public class CodeDxBuildAction implements Action, SimpleBuildStep.LastBuildAction, StaplerProxy {
```

Comment:
This class along with some others had the `Serializable` interface removed to satisfy findbugs maven prereq. I couldn't find any mention of needing `Serializable` on these classes in docs, and the plugin behaved the same after removing it. I'd assume `Serializable` is required for some types since jenkins stores our build status results for our graph etc. but this works fine even after restarts.